src/systems/audio-debug-system.js

```diff
@@ -23,10 +20,16 @@ AFRAME.registerSystem("audio-debug", {
   },
 
   init() {
+    this.max_debug_sources = 64;
```

Comment:
For consistent naming rule, `this.maxDebugSources` would be better?

Reply:
Yep, can do, I think I've spent way too long in Python land 😅